### ISSUE TITLE
BoundingVolumeMaterial Drawing 

### ISSUE THREAD

**Issue by Jrendl** [illusioneering/GopherGfx#1]
When applying the ``BoundingVolumeMaterial`` class to an object which has been re-posiotioned from the origin, the material doesn't show up at the same location that the object exists. I believe that this is because the class uses the ``Line3`` object and ``createFromBox`` function to visualize the outline of the bounding box. Line3 doesn't update it's position with the position of the box in question. I think that this class should have functionality built in to move the outline to the correct location.

**Comment by evansumarosenberg** [comment]
Yes, you are correct.  This was due to the fact that the Line3 class did not compute a new bounding box after the createFromBox function was executed.  This has already been fixed for future assignments, but it won't be applied retroactively for previous assignments unless you run `npm update` to pull the pull the latest version of GopherGfx.
